### PR TITLE
Add a simplified version of  slurm to batch script supporting multinodes.

### PR DESCRIPTION
**Pull request by zetxqx**
The generated template is under slurm-to-batch/examples.

Tested by submitting a Batch job at slurm-to-batch/examples/exmample_batch_job.yaml.
It is generated by replacing the placeholder in `example_batch_job_template.yaml`.

The job has two nodes and each node has 2 GPUs attached. The workload is printing the UUID of the GPU.
Here is the result: 

image

### CODE REVIEW

File: slurm-to-batch/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Usage
+   ```bash
+   python3 convert_slurm_batch_job.py <slurm_script_path> <batch_template_folder>
+   ```
+   * <slurm_script_path>: Path to your Slurm script.
+   * <batch_tempalte_folder> (Optional): Desired output directory for the template files(default to the Slurm script's parent folder).
+The script will generate two files: one in YAML format and one in JSON format. We recommend using the YAML file for better readability.
+
+## Template Placholders
```
Comment (aaronegolden) [inline slurm-to-batch/README.md:26]: typo: Placholders-->Placeholders

File: slurm-to-batch/examples/example_batch_job.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                  exit 0
+              else
+                  echo "slurmd not found, retrying in $WAIT_TIME seconds..."
+                  sleep $WAIT_TIME
+              fi
+          done
+
+          echo "slurmd did not start after $RETRIES attempts."
+          exit 1
+    - barrier: {}
```
Comment (aaronegolden) [inline slurm-to-batch/examples/example_batch_job.yaml:116]: I'd recommend naming the barriers just for readability, like

- barrier:
    name: slurm-started-all-vms

**Review by aaronegolden — APPROVED** [review]
LGTM in general. After comparing the YAML vs. JSON templates I'm tempted to say we should just do the YAML versions and leave out the JSON, since YAML handles multi-line scripts so much easier. Approved mod a couple typos anyway.

File: slurm-to-batch/README.md (hunk 1 above)
Comment (zetxqx) [reply slurm-to-batch/README.md]: updated

File: slurm-to-batch/examples/example_batch_job.yaml (hunk 2 above)
Comment (zetxqx) [reply slurm-to-batch/examples/example_batch_job.yaml]: updated

**Comment by zetxqx** [comment]
Removed the generated JSON batch template leaving only YAML format as well.

**Pull request merged**